### ISSUE TITLE
I can't switch between tabs with <TAB>

### ISSUE THREAD

**Issue by bitcodr** [wsdjeg/SpaceVim#4005]
## Expected behavior, english is required

I can't switch between tabs/window with TAB key

## The reproduce ways from Vim starting (Required!)

Just pressing TAB to switching between tabs/windows

## Debug info

Please press <kbd>SPC h I</kbd>, debug info will be put into clipboard, then paste all content below.

## Environment Information

- OS: linux
- vim version: -
- neovim version: 0.4.4
- SpaceVim version: 1.6.0-dev
- SpaceVim status: e3f18199

```
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   config/plugins/coc.vim

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	coc-settings.json
	sessions/

no changes added to commit (use "git add" and/or "git commit -a")
```

## Output of the `:SPDebugInfo!`

<details><summary> SpaceVim debug information </summary>

### SpaceVim options :

```toml
  auto_disable_touchpad = 1
  autocomplete_method = 'coc'
  autocomplete_parens = 1
  automatic_update = 1
  autowrite = 1
  bootstrap_before = 'myspacevim#before'
  buffer_index_type = 4
  checkinstall = 1
  colorcolumn = 120
  colorscheme = 'gruvbox'
  colorscheme_bg = 'dark'
  colorscheme_default = 'desert'
  commandline_prompt = '➭'
  custom_color_palette = []
  custom_plugins = [['sebdah/vim-delve', {'name': 'sebdah/vim-delve', 'merged': 0}], ['AndrewRadev/splitjoin.vim', {'name': 'AndrewRadev/splitjoin.vim', 'merged': 0}]]
  data_dir = '/home/bitcodr/.cache/'
  debug_level = 1
  default_indent = 2
  disabled_plugins = []
  enable_ale = 0
  enable_bepo_layout = 0
  enable_cursorcolumn = 0
  enable_cursorline = 1
  enable_debug = 0
  enable_googlesuggest = 0
  enable_guicolors = 1
  enable_insert_leader = 1
  enable_javacomplete2_py = 0
  enable_key_frequency = 0
  enable_language_specific_leader = 1
  enable_neocomplcache = 0
  enable_neomake = 1
  enable_os_fileformat_icon = 0
  enable_powerline_fonts = 1
  enable_projects_cache = 1
  enable_statusline_bfpath = 0
  enable_statusline_mode = 1
  enable_statusline_tag = 1
  enable_tabline_ft_icon = 0
  enable_vimfiler_filetypeicon = 0
  enable_vimfiler_gitstatus = 1
  enable_vimfiler_welcome = 1
  enable_ycm = 0
  error_symbol = '✖'
  escape_key_binding = 'jk'
  expand_tab = 1
  filemanager = 'vimfiler'
  filetree_direction = 'right'
  filetype_icons = {}
  force_global_config = 0
  gitcommit_issue_icon = ''
  gitcommit_pr_icon = ''
  github_username = ''
  go_metalinter_autosave = 1
  go_metalinter_command = 'golangci-lint'
  go_metalinter_enabeld = ['deadcode', 'errcheck', 'gosimple', 'govet', 'staticcheck', 'typecheck', 'unused', 'varcheck']
  go_rename_command = 'gopls'
  guifont = 'Sauce Code Pro Nerd Font Complete Mono:h22'
  hiddenfileinfo = 1
  home_files_number = 6
  hosts_url = 'https://raw.githubusercontent.com/racaljk/hosts/master/hosts'
  info_symbol = 'ⓘ'
  keep_server_alive = 1
  language = ''
  leader_guide_default_group_name = ''
  leader_guide_flatten = 1
  leader_guide_hspace = 5
  leader_guide_max_size = 0
  leader_guide_position = 'botright'
  leader_guide_run_map_on_popup = 1
  leader_guide_sort_horizontal = 0
  leader_guide_submode_mappings = {'<C-C>': 'win_close'}
  leader_guide_vertical = 0
  lint_on_save = 1
  lint_on_the_fly = 1
  max_column = 110
  plugin_bundle_dir = '/home/bitcodr/.cache/vimfiles/'
  plugin_manager = 'dein'
  plugin_manager_processes = 16
  plugin_name = 'fzf-neoyank'
  project_rooter_automatically = 1
  project_rooter_outermost = 1
  project_rooter_patterns = ['.git/', '_darcs/', '.hg/', '.bzr/', '.svn/', 'package.json', '.SpaceVim.d/']
  projects_cache_num = 20
  realtime_leader_guide = 1
  rehash256 = 1
  relativenumber = 1
  search_tools = ['rg', 'ag', 'pt', 'ack', 'grep', 'findstr', 'git']
  shiftwidth = 2
  sidebar_direction = ''
  sidebar_width = 30
  smartcloseignoreft = ['tagbar', 'vimfiler', 'defx', 'SpaceVimRunner', 'SpaceVimREPL', 'SpaceVimQuickFix', 'HelpDescribe', 'VebuggerShell', 'VebuggerTerminal', 'SpaceVimTabsManager']
  smartcloseignorewin = ['__Tagbar__', 'vimfiler:default']
  snippet_engine = 'neosnippet'
  spacevim_enable_vimfiler_filetypeicon = 1
  spacevim_max_column = 110
  src_root = 'E:\sources\'
  statusline_inactive_separator = 'bar'
  statusline_iseparator = 'nil'
  statusline_left_sections = ['winnr', 'major mode', 'filename', 'minor mode lighters', 'version control info', 'vcs', 'hunks']
  statusline_right_sections = ['cursorpos', 'percentage', 'input method', 'date', 'time']
  statusline_separator = 'arrow'
  statusline_unicode_symbols = 1
  terminal_cursor_shape = 2
  todo_labels = ['@fixme', '@question', '@todo', '@idea']
  update_retry_cnt = 3
  updatetime = 100
  version = '1.6.0-dev'
  vim_help_language = 'en'
  vimcompatible = 0
  warning_symbol = '⚠'
  wildignore = '*/tmp/*,*.so,*.swp,*.zip,*.class,tags,*.jpg,*.ttf,*.TTF,*.png,*/target/*,.git,.svn,.hg,.DS_Store,*.svg'
  windows_index_type = 3
  windows_leader = 's'
  windows_smartclose = 'q'
```

### SpaceVim layers :

```toml
[[layers]]
  name="autocomplete"
  auto_completion_return_key_behavior='complete'
  auto_completion_tab_key_behavior='smart'
[[layers]]
  name="checkers"
[[layers]]
  name="format"
  format_on_save=1
[[layers]]
  name="edit"
[[layers]]
  name="ui"
[[layers]]
  name="core"
[[layers]]
  name="core#banner"
[[layers]]
  name="core#statusline"
[[layers]]
  name="core#tabline"
[[layers]]
  name="VersionControl"
[[layers]]
  name="debug"
[[layers]]
  name="shell"
  default_position='bottom'
  default_height=30
[[layers]]
  name="git"
  git-plugin='fugitive'
[[layers]]
  name="lang#dockerfile"
[[layers]]
  name="lang#html"
[[layers]]
  name="lang#vim"
[[layers]]
  name="lang#markdown"
[[layers]]
  name="lang#sh"
[[layers]]
  name="lang#go"
[[layers]]
  name="lang#typescript"
[[layers]]
  name="lang#javascript"
  auto_fix=1
  enable_flow_syntax=1
[[layers]]
  name="tmux"
[[layers]]
  name="lsp"
  override_cmd={'go': ['rename', 'gopls', 'go-langserver', '-mode', 'stdio'], 'c': ['ccls', '--log-file=/tmp/ccls.log'], 'php': ['intelephense', '--stdio'], 'cpp': ['ccls', '--log-file=/tmp/ccls.log']}
  filetypes=['rust', 'go', 'javascript', 'vue', 'php', 'css', 'sh']
[[layers]]
  name="test"
  override_config={'rust_runner': 'cargotest', 'go_runner': 'delve'}
[[layers]]
  name="colorscheme"
[[layers]]
  name="default"
[[layers]]
  name="sudo"
[[layers]]
  name="tools"
[[layers]]
  name="fzf"
[[layers]]
  name="foldsearch"
```

### SpaceVim Health checking :

SpaceVim clipboard support check report:
Checking +clipboard:
      SUCCEED!
SpaceVim environment check report:
Current progpath: nvim(/usr/bin/nvim)
version: 800
OS: linux
[shell, shellcmdflag, shellslash]: ['/bin/bash', '-c', 0]
SpaceVim lua support check report:
Checking +lua:
      Failed : Known issue, neovim do not support lua now.
SpaceVim python support check report:
Checking +python3:
      SUCCEED!
Checking +python:
      SUCCEED!

### SpaceVim runtime log :

```log
[ SpaceVim ] : logger file  does not exists, only log for current process will be shown!
[ SpaceVim ] [23:42:08] [ Info ] Startup with no argv, current dir is used: /var/www/
[ SpaceVim ] [23:42:08] [ Info ] Can not find project local config, start loading global config
[ SpaceVim ] [23:42:08] [ Info ] global_dir is: /home/bitcodr/.SpaceVim.d/
[ SpaceVim ] [23:42:08] [ Info ] start to apply config [glob]
[   rooter ] [23:42:09] [ Info ] Load projects cache from: /home/bitcodr/.cache/SpaceVim/projects.json
[   rooter ] [23:42:09] [ Info ] Start to find root for: /var/www/
[   rooter ] [23:42:09] [ Info ]         (.git/):/var/www/
[   rooter ] [23:42:09] [ Info ]         (package.json):/var/www/
[   rooter ] [23:42:09] [ Info ] buffer name: No Name
[   rooter ] [23:42:09] [ Info ] same as current directory, no need to change.
[ SpaceVim ] [23:42:09] [ Info ] try to open SpaceVim welcome page
[   rooter ] [23:42:09] [ Info ] buffer name: No Name
[   rooter ] [23:42:09] [ Info ] same as current directory, no need to change.
[   rooter ] [23:42:11] [ Info ] Start to find root for: /var/www/vimfiler:default
[   rooter ] [23:42:11] [ Info ]         (.git/):/var/www/
[   rooter ] [23:42:11] [ Info ]         (package.json):/var/www/
[   rooter ] [23:42:11] [ Info ] buffer name: vimfiler:default
[   rooter ] [23:42:11] [ Info ] same as current directory, no need to change.
[   rooter ] [23:42:11] [ Info ] buffer name: No Name
[   rooter ] [23:42:11] [ Info ] same as current directory, no need to change.
[ SpaceVim ] [23:42:14] [ Info ] key binding guide float statusline winid:1004
[   rooter ] [23:42:15] [ Info ] buffer name: No Name
[   rooter ] [23:42:15] [ Info ] same as current directory, no need to change.
[ SpaceVim ] [23:42:15] [ Info ] close float statusline winid:1004
[ SpaceVim ] [23:42:15] [ Info ] key binding guide float statusline winid:1006
[   rooter ] [23:42:15] [ Info ] buffer name: No Name
[   rooter ] [23:42:15] [ Info ] same as current directory, no need to change.
[ SpaceVim ] [23:42:15] [ Info ] close float statusline winid:1006
[   rooter ] [23:42:17] [ Info ] Start to find root for: /tmp/nvim5Cj6uM/3/issue_report.md
[   rooter ] [23:42:17] [ Info ] buffer name: /tmp/nvim5Cj6uM/3/issue_report.md
[   rooter ] [23:42:17] [ Info ] same as current directory, no need to change.
```
</details>

## Screenshots

If you have any screenshots for this issue please upload here. BTW you can use https://asciinema.org/ for recording video  in terminal.

**Comment by Rom1deTroyes** [comment]
Please, look at #3347 (and #3940 #3823 #3818 ...).

**Comment by wsdjeg** [comment]
In old issue, I have said, the `<tab>` key binding is same as `ctrl-i` in vim, it is use for jump between jump list, together with `ctrl-o`. so we should not override it. If you want to use `<tab>` to jump between windows, just create a key binding mapping in bootstrap function.

```
nnoremap <tab> <C-w>w
```

**Comment by ryuheechul** [comment]
This change was one of the putting off and surprising change of behaviour when I bumped to `v1.5.0+`. (because I didn't know why it suddenly stopped working at first but now I can see why it's changed)

However I still want it to work in the old way, so thanks for @wsdjeg 's answer above.

On top of that, In my case I looked it up what's the actual difference between old `v1.4.0` and now.
Which is the absence of [this line](https://github.com/SpaceVim/SpaceVim/blob/v1.4.0/autoload/SpaceVim.vim#L1164).

So I just added it into `myspacevim#before` function like [this](https://github.com/ryuheechul/dotfiles/commit/f7c0d1c66d79849e3447845587a85059d9c92f57#diff-af345363502322c435451a8ccfdbfe68f5f23566401aeceb64a9eb56225d4c78) and it just works like just as before. 😃
